blocks/header/header.css

```diff
@@ -1,809 +1,268 @@
-
-  header nav .tools ul li img {
-    width: 14px;
-    height: 14px;
-    object-fit: contain;
-  }
-
-  /* search */
-  header nav .search {
-    align-self: start;
-    justify-self: start;
-    position: initial;
-    grid-area: search;
-    height: 50px;
-    width: 100%;
-    padding: 0;
-    margin: 0;
-  }
-
-  header nav .search div {
-    padding: 10px 0 0;
-    display: flex;
-    flex-flow: row;
-    justify-content: right;
-  }
-
-  header nav .search input {
-    position: relative;
-    display: block;
-    padding: 10px 35px 10px 15px;
-    width: 290px;
-    height: 40px;
-    color: var(--c-grey-4);
-    border: 1pt solid var(--c-grey-1);
-    border-radius: 0;
-    background-color: #fff;
-    box-shadow: none;
-    outline: none;
-    font-size: 12px;
-  }
-
-  header nav .search input:focus {
-    border: 1pt solid black;
-  }
-
-  header nav .search .search-button {
-    height: 40px;
-    position: absolute;
-    color: #bbb;
-    border-radius: 0;
-    background-color: transparent;
-    font-size: 18px;
-    z-index: 99;
-  }
-
-  /* sections */
-  header nav .sections {
-    display: block;
-    position: unset;
-    overflow: unset;
-    grid-area: sections;
-    height: 48px;
-    width: 100%;
-    background-color: var(--c-grey-1);
-  }
-
-  header nav .sections .sections-list {
-    position: relative;
-    display: flex;
-    flex-flow: row;
-    align-items: center;
-    width: calc(93% - 30px);
-    margin: 0 auto;
-  }
-
-  header nav .sections * {
-    font-size: 12px;
-  }
-
-  header nav .sections .section {
-    display: block;
-  }
-
-  header nav .sections .section > a {
-    border: 0;
-    padding: 15px 24px;
-    height: 48px;
-    cursor: pointer;
+    justify-content: center;
   }
 
-  header nav .sections .section.expand {
+  .header__nav {
     padding: 0;
+    max-width: 1200px;
   }
 
-  header nav .sections .section > a:hover,
-  header nav .sections .section.expand > a {
-    background-color: white;
-  }
-
-  header nav .sections .section > a::after {
-    position: initial;
-    display: inline-block;
-    width: 1px;
-    content: '\f105';
-    font-size: 12px;
-    margin-left: 6px;
-  }
-
-  header nav .sections .section.expand a::after {
-    content: '\f107';
+  .header__link:hover,
+  .header__link:focus {
+    border-bottom: solid 2px var(--c-grey-200);
   }
 
-  header nav .sections .section .overview {
-    display: none;
+  .header__link:any-link:active {
+    border-bottom: solid 2px var(--c-main-black);
   }
 
-  /* flyout */
-  header nav .sections .section > ul {
-    position: absolute;
-    left: 0;
+  .header__action-link.header__hamburger-menu {
     display: none;
-    background-color: #fff;
-    padding: 40px 0 0 92px;
-    width: 100%;
-    z-index: 1010;
   }
 
-  header nav .sections .section.expand .navigation-cta {
-    margin: 0 0 0 -92px !important;
-    width: calc(100% + 92px) !important;
-    padding: 16px 0;
+  .header__actions-list,
+  .header__main-links {
```

Comment:
Do we need this ? I see that `display:flex` is set at 122

Reply:
Yup, it is not needed anymore - removed